src/floating-focus.spec.js

```diff
@@ -323,6 +323,62 @@ describe('Floating focus', () => {
 		expect(floater.style.height).toBe(`${rect.height}px`);
 	});
 
+	it('Should automatically reposition the \'floater\' when the target element\'s position changes', async () => {
+		const floatingFocus = new FloatingFocus();
+		const target = document.createElement('div');
+		document.body.appendChild(target);
+
+		const rect = {
+			left: 42,
+			top: 84,
+			width: 42,
+			height: 128
+		};
+
+		target.getBoundingClientRect = jest.fn().mockImplementation(() => rect);
+
+		floatingFocus.handleKeyDown({keyCode: 9});
+		floatingFocus.enableFloatingFocus();
+		floatingFocus.handleFocus({target}, true);
+
+		expect(floatingFocus.floater.style.height).toBe(`${rect.height}px`);
+
+		await new Promise(resolve => setTimeout(resolve, 250));
```

Comment:
https://jestjs.io/docs/en/timer-mocks

Reply:
Ja de reden dat ik dat hier (vooralsnog) niet gebruikt heb is dat in de rest van de tests ook gewoon met normale timeouts wordt gewerkt, dus met al die async tests gaat deze geen verschil maken

Reply:
Aan de andere kant kan ik het ook gewoon overal in het bestand vervangen, guess I'll do that :)